tests/admin_docs/tests.py

```diff
@@ -368,6 +386,31 @@ def test_model_docstring_renders_correctly(self):
         self.assertContains(self.response, body, html=True)
         self.assertContains(self.response, model_body, html=True)
 
+    def test_model_detail_title(self):
+        self.assertContains(
```

Comment:
one line is probably fine for these assertions (longer lines up to 119 chars are preferred if it helps readability)
Is this a missing test? It's a bit confusing to add it as part of this change. You could make it a separate commit.